### PR TITLE
Configure BGP peering How to

### PR DESCRIPTION
**Pull request by bmckercher123**
Add new How to for Networking

- Can merge at any time
- No need to back port
- Reviewed by Casey and Brendan

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 7ebfb7534425a7486ad54be30f50f7efa063f3f7

https://deploy-preview-3047--calico.netlify.com

File: master/networking/bgp.md [hunk 1]
```diff
@@ -1,424 +1,172 @@
...
+For a deeper look at common on-premises deployment models, see [Calico over IP Fabrics]({{site.baseurl}}/{{page.version}}/reference/architecture/design/l2-interconnect-fabric).
 
-1. If the resource _does_ exist, use the following command to retrieve it and save it
-   to a file.
+### Before you begin...
 
-    ```
-    calicoctl get bgpconfig default --export -o yaml > bgp.yaml
-    ```
+`calicoctl` must be installed and configured.
```
Comment (caseydavenport) [inline master/networking/bgp.md]: Should we link to the calicoctl getting started page here?

File: master/networking/bgp.md [hunk 2]
```diff
@@ -1,424 +1,172 @@
...
-#### Example
+#### Change AS number for a particular node
 
-To check the status of the peerings on {{site.prodname}} node `"node1"`, SSH into
-`"node1"` and run the following command.
+You can configure an AS for a particular node by modifying the node object using `calicoctl`. For example, the following command changes the node named **node-1** to belong to **AS 64514**.
 
 ```
-sudo calicoctl node status
+calicoctl patch node node-1 -p '{"spec": {"bgp": {“asNumber”: “64513”}}}'
```
Comment (caseydavenport) [inline master/networking/bgp.md]: woops, this 64513 should be a 64514

File: master/networking/bgp.md (hunk 1 above)
Comment (bmckercher123) [reply master/networking/bgp.md]: Yep, added link.

File: master/networking/bgp.md (hunk 2 above)
Comment (bmckercher123) [reply master/networking/bgp.md]: Fixed

**Pull request merged**